plugins/generator-1.19.2/forge-1.19.2/templates/particle.java.ftl

```diff
@@ -117,6 +117,7 @@ package ${package}.client.particle;
 		</#if>
 
 		<#if hasProcedure(data.additionalExpiryCondition)>
+		Level world = this.level;
 		double x = this.x;

```

Comment:
Please also remove those `double x = this.x;` etc. as they are redundant while are you at it